excludes.txt

```diff
@@ -4,6 +4,11 @@ hudson.matrix.AxisTest
 # TODO until we drop support for 2.249.x, work around https://github.com/jenkinsci/git-plugin/pull/1093
 jenkins.plugins.git.ModernScmTest
 
+# TODO until we drop support for 2.249.x, work around https://github.com/jenkinsci/git-client-plugin/pull/675 not available in git client plugin 3.6.0
+org.jenkinsci.plugins.gitclient.CliGitAPIImplTest.test_list_branches
+org.jenkinsci.plugins.gitclient.JGitAPIImplTest.test_list_branches
+org.jenkinsci.plugins.gitclient.JGitApacheAPIImplTest.test_list_branches
```

Comment:
I suspect you will need to
```suggestion
org.jenkinsci.plugins.gitclient.CliGitAPIImplTest
org.jenkinsci.plugins.gitclient.JGitAPIImplTest
org.jenkinsci.plugins.gitclient.JGitApacheAPIImplTest
```
since I do not think Surefire is able to exclude individual test cases. (This is the downside of using an exclusion file to _skip running_ test suites. The original system ran everything and then deleted `TEST-*.xml` with known failures, which also operated at the suite level but at least _could_ have been improved to use the likes of `xmlstarlet` to patch out individual cases.)